locust.md

```diff
@@ -1,134 +0,0 @@
-# Quickly setting up a load test client using Locust
```

Comment:
Can we not delete this file? I already shared the `https://github.com/lichenran1234/load-test/blob/main/locust.md` link to some customers (my fault), and that link will be invalidated if this file is removed.

Instead, let's keep this file but modify the content to be `Please refer to README.md`?

Reply:
Makes sense. I added it back in.